docs/groovy-script/mods/pedestalcrafting/pedestal_crafting.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Pedestal Crafting"
+titleTemplate: "Pedestal Crafting | CleanroomMC"
+description: "Converts a variable number of input items into a single output item. The recipe can be customized with various particles."
+source_code_link: "https://github.com/Ender-Development/PedestalCrafting-Patched/blob/master/src/main/java/me/axieum/mcmod/pedestalcrafting/compat/groovyscript/PedestalCrafting.java"
+---
+
+# Pedestal Crafting (Pedestal Crafting)
+
+## Description
+
+Converts a variable number of input items into a single output item. The recipe can be customized with various particles.
+
+## Identifier
+
+Refer to this via any of the following:
+
+```groovy:no-line-numbers {1}
+mods.pedestalcrafting.pedestal_crafting/* Used as page default */ // [!code focus]
+mods.pedestalcrafting.pedestalcrafting
+mods.pedestalcrafting.pedestalCrafting
+mods.pedestalcrafting.PedestalCrafting
+```
+
+
+## Adding Recipes
+
+### Recipe Builder
+
+Just like other recipe types, the Pedestal Crafting also uses a recipe builder.
+
+Don't know what a builder is? Check [the builder info page](../../getting_started/builder.md) out.
+
+:::::::::: details mods.pedestalcrafting.pedestal_crafting.recipeBuilder() {open id="abstract"}
+- `IngredientList<IIngredient>`. Sets the item inputs of the recipe. Requires greater than or equal to 2.
```

Comment:
in this case, it seems more reasonable to have a field like `IIngredient center`, as currently `input` is being used for both `core` and `inputs` in when initializing `PedestalRecipe`.